Src/FluentAssertions.Json/JTokenDifferentiator.cs

```diff
@@ -74,7 +82,7 @@ private static Difference CompareExpectedItems(JArray actual, JArray expected, J
                 bool match = false;
                 for (int actualIndex = matchingIndex; actualIndex < actualChildren.Length; actualIndex++)
                 {
-                    var difference = FindFirstDifference(actualChildren[actualIndex], expectedChild, true, config);
```

Comment:
`CompareExpectedItems` is only called when `ignoreExtraProperties` is `true`, so the three `true` literals in this method always corresponds to the value of `ignoreExtraProperties`.